index.html

```diff
@@ -2,6 +2,16 @@
 	<head>
 	</head>
 	<body>
-	Hello!
+		Hello!
```

Comment:
1. new folder for task
2. wrong indentations, 2 spaces is correct
3. empty line at the EOF in each file, you see red sign